app/controllers/customers_controller.rb

```diff
@@ -0,0 +1,15 @@
+class CustomersController < ApplicationController 
+
+  def show
+    @customer = Customer.find(params[:id])
+  end
+
+  def update
```

Comment:
A more RESTful option here would be to pull this functionality into a separate controller - the CustomerItemsController. There, you can add a `create` action that will do this logic. Typically, we want the `update` here to just update attributes for the customer rather than adding joins records associating the customer to the items.